.github/actions/action-lacework-scan/action.yml

```diff
@@ -0,0 +1,130 @@
+name: 'Run Lacework image scan'
+description: 'Scan the provided image with Lacework scan tool and return a JSON with the results.'
+
+inputs:
+  image:
+    description: 'Full URL of the image to scan.'
+    required: true
+  docker-username:
+    description: 'Docker username.'
+    required: true
+  docker-password:
+    description: 'Docker password.'
+    required: true
+  lw-account-name:
+    description: 'Lacework account name.'
+    required: true
+  lw-access-token:
+    description: 'Lacework access token.'
+    required: true
+  severity-filter:
+    description: 'Specify severities of CVEs to be included in the result, in CSV format, e.g. "Critical,High,Medium,Low,Info". (Default: "Critical,High")'
+    default: 'Critical,High'
+  save-results-in-lacework:
+    description: 'Save results to Lacework. (Default: false)'
+    default: 'false'
+  debug-mode:
+    description: 'Set to "true" to enable debug mode. It generates extra artifacts with partial results. (Default: false)'
+    default: 'false'
```

Comment:
optional: Maybe this could use the `ACTIONS_STEP_DEBUG` secret that actions use as a [standard variable](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging#enabling-step-debug-logging) for debugging. That way there's no code change to trigger debugging, just adding a secret.
Not sure if it's even accessible inside a composite action though.

Reply:
Thanks, I didn't know that. Changed to use `ACTIONS_STEP_DEBUG` for debugging.